### ISSUE TITLE
Verify default bookmark creation time against reference machine

### ISSUE THREAD

**Issue by piatra** [mozilla/activity-stream#3633]
See more https://github.com/mozilla/activity-stream/pull/3623#discussion_r142246352

Investigate timestamp for default bookmarks on reference hardware.

**Comment by Mardak** [comment]
The reference hardware is about the median user hardware, so we should probably be more conservative than that. I suppose we could just put in a large-enough number that hopefully doesn't cause actual bookmarks to be hidden. It doesn't seem too likely that a user will create a Firefox profile and immediately bookmark within the first X seconds, although we probably also want to be careful of users importing data…

**Comment by piatra** [comment]
I've done a test importing bookmarks into a fresh FFx profile. The bookmark dateAdded is not the the one from the original browser, it's the moment you add it in FFx.

**Comment by Mardak** [comment]
Right, the concern is that imports can happen early near profile creation time. Either via the wizard or the manual migration.

**Comment by piatra** [comment]
I've installed Nightly on an older/slower machine and got a delta of 32s. The slow-ness might be due to the fact that this profile was created while installing Firefox.

While looking into this I also came across another related issue. I've been creating new profiles by going to `about:profiles`. It seems that when you create a new profile from that page only a `times.json` file is created which contains the timestamp returned by `ProfileAge.created`.
The bookmarks are added when starting the profile for the first time which can be much later.

Is this the usual way to do it? Are there any other ways to create profiles?
